src/Filtering/ImageFeature/CMakeLists.txt

```diff
@@ -33,3 +33,5 @@ add_example(FindZeroCrossingsInSignedImage)
 #===================================
 
 add_example(AdditiveGaussianNoiseImageFilter)
+
+add_example(DerivativeImage)
```

Comment:
Could a baseline check please be added to verify against previous output data? See [https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/blob/master/src/Filtering/ImageFeature/CMakeLists.txt#L2](https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/blob/master/src/Filtering/ImageFeature/CMakeLists.txt#L2) for example syntax.